README.md

```diff
@@ -42,3 +42,18 @@ This is an application that I wrote long ago, but have since repurposed to play
 
 - Tests can be run using `rake test`
 - For test coverage, run your tests, open up `coverage/index.html` in your browser
+
+## Docker Deploy
+
+* [Install docker](https://docs.docker.com/engine/installation/)
+* [Install docker-compose](https://docs.docker.com/compose/install/)
+
+Run the following three commands in the project's root directory:
+
+```
+docker-compose build
+docker-compose run web rake db:setup
```

Comment:
This command failed for me the first time, with this message:
![screen shot 2017-01-25 at 15 23 32](https://cloud.githubusercontent.com/assets/2058614/22307865/ba20cdba-e312-11e6-91cb-e27001547814.png)

However, running the same command a second time worked. Any guesses about what's causing that?